### ISSUE TITLE
 IOS 14 - Xcode 12 > xctest client exit with code: 65, signal: null

### ISSUE THREAD

**Issue by mariushorvat** [alibaba/macaca#985]
Hi guys, when I run: app-inspector -u xxxx --verbose on IOS 14 - Xcode 12
Thanks!

xctest-client.js:222:18 [master] pid:17443 Testing failed:
Module compiled with Swift 5.2 cannot be imported by the Swift 5.3 compiler: /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS/Swifter.framework/Modules/Swifter.swiftmodule/x86_64-apple-ios-simulator.swiftmodule
Testing cancelled because the build failed.

** TEST FAILED **

The following build commands failed:
CompileSwift normal x86_64 /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift
CompileSwift normal x86_64 /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDAlertController.swift
CompileSwift normal x86_64 /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDElementController.swift
CompileSwift normal x86_64 /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDScreenshotController.swift
CompileS...

xctest-client.js:233:20 [master] pid:17443 please check project: /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD.xcodeproj
xctest-client.js:329:14 [master] pid:17443 killing runnerProc pid: 17479
xctest-client.js:245:16 [master] pid:17443 xctest client exit with code: 65, signal: null

**Comment by xudafeng** [comment]
https://github.com/macacajs/app-inspector/issues/254

**Comment by paradite** [comment]
Sorry I was busy with other things, taking a look this week.

Like I said in https://github.com/macacajs/app-inspector/issues/254, you can manually compile `XCTestWD` Carthage dependencies and replace the old dependencies copied over from `xctestwd-frameworks` with new ones. the `xctestwd-frameworks` is merely an installation-step optimization to pre-compile the Carthage dependencies and save some time. Unfortunately it needs to be updated with every Swift update.

**Comment by paradite** [comment]
fixed, please remove and reinstall relevant packages

**Comment by tking16** [comment]
After runnining `npm uninstall app-inspector` and then `npm i app-inspector -g`

I get this during installation
```
CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDAccessibility.swift (in target 'XCTestWD' from project 'XCTestWD')
    cd /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDDispatch.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDServer.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDAlertController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDElementController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDScreenshotController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSessionController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSourceController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDTitleController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDUrlController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDWindowController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDAlert.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDErrors.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDReponse.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDFailureProofTestCase.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDSession.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDStatus.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCUIElementTypeTransformer.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDFindElementUtils.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDMathUtils.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDXPath.swift -primary-file /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDAccessibility.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDApplicationTree.swift -emit-module-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility\~partial.swiftmodule -emit-module-doc-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility\~partial.swiftdoc -emit-module-source-info-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility\~partial.swiftsourceinfo -serialize-diagnostics-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility.dia -emit-dependencies-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility.d -emit-reference-dependencies-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility.swiftdeps -target x86_64-apple-ios9.0-simulator -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk -I /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator -I /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/lib -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/Frameworks -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/PrivateFrameworks -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/Developer/Library/Frameworks -enable-testing -g -import-underlying-module -module-cache-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/ModuleCache.noindex -swift-version 4 -enforce-exclusivity\=checked -Onone -D DEBUG -serialize-debugging-options -Xcc -working-directory -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD -enable-anonymous-context-mangled-names -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/swift-overrides.hmap -Xcc -iquote -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-generated-files.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-own-target-headers.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/all-product-headers.yaml -Xcc -iquote -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-project-headers.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator/include -Xcc -I/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/usr/include/libxml2 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources-normal/x86_64 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources/x86_64 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources -Xcc -DXCTESTWD_PORT\=8001 -Xcc -ivfsoverlay -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/unextended-module-overlay.yaml -target-sdk-version 14.0 -module-name XCTestWD -o /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDAccessibility.o -index-store-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Index/DataStore -index-system-modules
/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift:9:8: error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler: /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS/Swifter.framework/Modules/Swifter.swiftmodule/x86_64-apple-ios-simulator.swiftmodule
import Swifter
       ^

CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDApplicationTree.swift (in target 'XCTestWD' from project 'XCTestWD')
    cd /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDDispatch.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDServer.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDAlertController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDElementController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDScreenshotController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSessionController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSourceController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDTitleController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDUrlController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDWindowController.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDAlert.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDErrors.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDReponse.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDFailureProofTestCase.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDSession.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDStatus.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCUIElementTypeTransformer.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDFindElementUtils.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDMathUtils.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDXPath.swift /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDAccessibility.swift -primary-file /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDApplicationTree.swift -emit-module-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree\~partial.swiftmodule -emit-module-doc-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree\~partial.swiftdoc -emit-module-source-info-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree\~partial.swiftsourceinfo -serialize-diagnostics-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree.dia -emit-dependencies-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree.d -emit-reference-dependencies-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree.swiftdeps -target x86_64-apple-ios9.0-simulator -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk -I /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator -I /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/lib -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/Frameworks -F /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/PrivateFrameworks -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/Developer/Library/Frameworks -enable-testing -g -import-underlying-module -module-cache-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/ModuleCache.noindex -swift-version 4 -enforce-exclusivity\=checked -Onone -D DEBUG -serialize-debugging-options -Xcc -working-directory -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD -enable-anonymous-context-mangled-names -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/swift-overrides.hmap -Xcc -iquote -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-generated-files.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-own-target-headers.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/all-product-headers.yaml -Xcc -iquote -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/XCTestWD-project-headers.hmap -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Products/Debug-iphonesimulator/include -Xcc -I/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/usr/include/libxml2 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources-normal/x86_64 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources/x86_64 -Xcc -I/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/DerivedSources -Xcc -DXCTESTWD_PORT\=8001 -Xcc -ivfsoverlay -Xcc /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/unextended-module-overlay.yaml -target-sdk-version 14.0 -module-name XCTestWD -o /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Build/Intermediates.noindex/XCTestWD.build/Debug-iphonesimulator/XCTestWD.build/Objects-normal/x86_64/XCTestWDApplicationTree.o -index-store-path /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/build/Index/DataStore -index-system-modules
/usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift:9:8: error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler: /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS/Swifter.framework/Modules/Swifter.swiftmodule/x86_64-apple-ios-simulator.swiftmodule
import Swifter
       ^

** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDAlertController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDElementController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDScreenshotController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSessionController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDSourceController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDTitleController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDUrlController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Controllers/XCTestWDWindowController.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDAlert.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDErrors.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDReponse.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/XCTestWDFailureProofTestCase.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDSession.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCTestWDStatus.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/XCUIElementTypeTransformer.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDFindElementUtils.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDMathUtils.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Extensions/XCTestWDXPath.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDAccessibility.swift
	CompileSwift normal x86_64 /usr/local/Cellar/node/15.0.1/lib/node_modules/app-inspector/node_modules/xctestwd/XCTestWD/XCTestWD/Server/Modules/Utils/XCTestWDApplicationTree.swift
	Compil
```

**Comment by kmilic** [comment]
>  error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler

@tking16 I had similar issue, it seems that fix is made for xcode 12.2 (based on this line ' error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler'). Worked for me after upgrading xcode 🙂

**Comment by paradite** [comment]
> > error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler
> 
> @tking16 I had similar issue, it seems that fix is made for xcode 12.2 (based on this line ' error: module compiled with Swift 5.3.1 cannot be imported by the Swift 5.3 compiler'). Worked for me after upgrading xcode 🙂

You are right. I was using the latest Xcode version 12.2, which is using Swift 5.3.1. So Swift 5.3 support is "skipped" in the sense.
